lib/components/app/responsive-webapp.js

```diff
@@ -122,71 +203,91 @@ class ResponsiveWebapp extends Component {
 // connect to the redux store
 
 const mapStateToProps = (state, ownProps) => {
+  const title = getTitle(state)
   return {
     activeItinerary: getActiveItinerary(state.otp),
+    activeSearchId: state.otp.activeSearchId,
     uiUrlParams: getUiUrlParams(state.otp),
     currentPosition: state.otp.location.currentPosition,
     query: state.otp.currentQuery,
+    searches: state.otp.searches,
     mobileScreen: state.otp.ui.mobileScreen,
     initZoomOnLocate: state.otp.config.map && state.otp.config.map.initZoomOnLocate,
-    modeGroups: state.otp.config.modeGroups
+    modeGroups: state.otp.config.modeGroups,
+    title
   }
 }
 
 const mapDispatchToProps = {
-  setLocationToCurrent,
-  setMapCenter,
-  setMapZoom,
-  getCurrentPosition,
+  clearActiveSearch,
+  clearLocation,
   findRoute,
   formChanged,
+  getCurrentPosition,
+  parseUrlQueryString,
   receivedPositionResponse,
+  setActiveItinerary,
+  setActiveSearch,
+  setLocationToCurrent,
   setMainPanelContent,
+  setMapCenter,
+  setMapZoom,
   setQueryParam,
-  parseUrlQueryString
+  setViewedRoute,
+  setViewedStop
```

Comment:
These actions are increasing quite a bit in number. Would it make more sense to combine some of the component handlers as actions and just import those large actions?